webpack.config.ts

```diff
@@ -1,32 +1,41 @@
 import path from "path";
-import {Configuration, DefinePlugin} from "webpack";
+import {Configuration, DefinePlugin, EnvironmentPlugin, WebpackOptionsNormalized} from "webpack";
 
-module.exports = (env, options): Configuration => ({
-	devtool: options.mode !== 'production' ? 'source-map' : undefined,
-	context: path.join(__dirname, '/extension'),
+import config from "./mv3-hot-reload.config";
+
+const isDev = (options: WebpackOptionsNormalized) => options.mode !== "production";
+
+const srcDir = path.join(__dirname, "extension");
+const tsSrcDir = path.join(srcDir, "ts");
+const getEntry = (name: string) => {
+	return [path.join(tsSrcDir, name), ...(isDev ? [`mv3-hot-reload/${name}`] : [])];
```

Comment:
`isDev` function is not being applied here

Reply:
oopsie